### PR TITLE
Refactor contracts

### PR DESCRIPTION
**Pull request by imollov**

### CODE REVIEW

File: contracts/DssCronKeeper.sol [hunk 1]
```diff
@@ -55,25 +55,25 @@ contract DssCronKeeper is KeeperCompatibleInterface, Ownable {
...
-        (bool success, ) = address(this).delegatecall(performData);
+    function performUpkeep(bytes calldata _performData) external override {
+        (bool success, ) = address(this).delegatecall(_performData);
         require(success, "failed to perform upkeep");
     }
 
     /**
      * @notice Execute cron job with params
-     * @dev Job work function checks if is still pending and still network turn,
+     * @dev The job work function checks if it's workable and if the network is in turn,
```
Comment (mradkov) [inline contracts/DssCronKeeper.sol:65]:
```suggestion
     * @dev The runJob function checks if it's workable and if the network is in turn,
```

Comment (mradkov) [reply contracts/DssCronKeeper.sol:65]: 🤔

Comment (imollov) [reply contracts/DssCronKeeper.sol:65]:
```suggestion
     * @dev The IJob work function checks if it's workable and if the network is in turn,
```

Does it make more sense like this?

Comment (mradkov) [reply contracts/DssCronKeeper.sol:65]: Yes, thanks!

**Review by mradkov — APPROVED** [review]

**Pull request merged**